src/op_mode/show_nat66_rules.py

```diff
@@ -42,9 +42,16 @@
         data = data_json[idx]
         
         # If there is no index 3, we don't think this is the record we need to check
+        # We need to filter the rule for Len (expr) <= 3 first, which is not what we should be concerned with
```

Comment:
What I'd like to see here is an explanation of _why_ such rules can exist in the nftables config. How do they appear there?

Can we use _only_ a check for required fields rather than for explicit field count?

Reply:
Let me see

Reply:
Maybe I can consider improving  field search logic. Let me see

Reply:
@dmbaturin done